manifests/08-report_overview_crd.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: report-overviews.external-pipeline.redhat.com
+spec:
+  group: external-pipeline.redhat.com
```

Comment:
Woudln't it make better sense to call the group for example "insights.redhat.com" ? This looks a bit dependent on one particular place. What do you think ?